src/textual/devtools/server.py

```diff
@@ -80,4 +80,4 @@ def _make_devtools_aiohttp_app(
 
 
 if __name__ == "__main__":
-    _run_devtools()
+    _run_devtools(True)
```

Comment:
This was a bug (missing positional param)

Reply:
Think this block is dead code. Could just be removed entirely.

Reply:
👍 I'll remove